src/components/AdjustableSidebarWidth.vue

```diff
@@ -113,6 +114,14 @@ export default {
       type: Boolean,
       default: false,
     },
+    disableResizing: {
```

Comment:
Also, I find it a little odd/funny that there's a component called `AdjustableSidebarWidth` that can be configured to _not_ be adjustable, but I completely understand how we got here, haha 😅

Not necessary now, but maybe we should rename the component now that it can be restricted in this way.

Reply:
hmm.. `DynamicSidebar` ? 🤣 `FlexibleSidebar` ??? 👀